samples/powerstore_v230_k8s_123.yaml

```diff
@@ -89,25 +95,30 @@ spec:
         # Default value: false
         - name: X_CSI_HEALTH_MONITOR_ENABLED
           value: "false"
-
       # nodeSelector: Define node selection constraints for node pods.
       # For the pod to be eligible to run on a node, the node must have each
       # of the indicated key-value pairs as labels.
       # Leave as blank to consider all nodes
       # Allowed values: map of key-value pairs
       # Default value: None
-      # Examples:
-      #   node-role.kubernetes.io/worker: ""
       nodeSelector:
-      #   node-role.kubernetes.io/worker: ""
```

Comment:
1. We shouldn't be changing the worker label. This change is needed only for master. 
2. Same comment for line 114.
3. Please check and revert in other files as well.

Reply:
@spriya-m This is just a example. We can kept either worker or master. To be consistent across other csi drivers I have changed to master again.